community/unsafe/src/main/java/org/neo4j/unsafe/impl/internal/dragons/FeatureToggles.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (c) 2002-2015 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.unsafe.impl.internal.dragons;
+
+import java.security.AccessController;
+import java.security.PrivilegedAction;
+
+public class FeatureToggles
```

Comment:
If its a new way of defining feature flags then it should contain proper javadoc, all old ways of defining those kind of properties should be updated and probably it should not be in `org.neo4j.unsafe.impl.internal.dragons` package